nix/pkgs/update-hie/default.nix

```diff
@@ -0,0 +1,5 @@
+{ writeShellScriptBin, gen-hie }:
+writeShellScriptBin "update-hie" ''
+  ${gen-hie}/bin/gen-hie --cabal > hie-cabal.yaml
```

Comment:
I haven't actually run this - does this actually print something "useful" ? Otherwise i think you might want to add something like `@echo Updating 'hie-cabal.yaml'` or so. Makes it easier to spot in longer outputs in case this maybe gets run as part of some script or so.

Reply:
None of the other scripts that update things print stuff :shrug: generally I think "silent unless there's a problem" is okay?